src/Cards.h

```diff
@@ -0,0 +1,112 @@
+
+#pragma once
+#ifndef EIGHT_MINUTE_EMPIRE_CARDS_H
+#define EIGHT_MINUTE_EMPIRE_CARDS_H
+
+#include <string>
+#include <ostream>
+#include <vector>
+#include <ctime>
+#include <algorithm>
+#include <iostream>
+using namespace std;
+
+using std::string;
+
+class Cards {
+public:
+    Cards();
+
+    Cards(const string &good, const string &action);
+
+    virtual ~Cards();
+
+    const string &getGood() const;
+
+    void setGood(const string &good);
+
+    const string &getAction() const;
+
+    void setAction(const string &action);
+
+    friend ostream &operator<<(ostream &os, const Cards &cards);
+
+public:
+
+
+
+
+private:
+    string good;
+    string action;
+};
+
+class Deck {
+    std::vector<Cards> deck;
```

Comment:
```suggestion
    std::vector<Cards> * deck;
```

This should be a pointer, I think